File: easydoc-javadoc/src/main/java/com/star/easydoc/view/inner/SupportView.java
package com.star.easydoc.view.inner;

import java.awt.*;

import javax.swing.*;

import com.intellij.openapi.ui.DialogWrapper;
import org.jetbrains.annotations.Nullable;

/**
 * @author wangchao
 * @date 2020/08/29
 */
public class SupportView extends DialogWrapper {

    private JLabel alipayLabel;
    private JLabel wechatLabel;
    private JPanel panel;

    public SupportView() {
        super(false);
        init();
        setTitle("感谢大佬！");
    }

    @Nullable
    @Override
    protected JComponent createCenterPanel() {
        return panel;
    }

    private void createUIComponents() {
        ImageIcon alipayIcon = new ImageIcon(this.getClass().getResource("/alipay.jpg"));
        double scale = 400d / alipayIcon.getIconHeight();
        alipayIcon.setImage(alipayIcon.getImage().getScaledInstance((int)(alipayIcon.getIconWidth() * scale), 400, Image.SCALE_DEFAULT));
        alipayLabel = new JLabel(alipayIcon);
        alipayLabel.setVisible(true);

        ImageIcon wechatIcon = new ImageIcon(this.getClass().getResource("/wechat.png"));
        scale = 400d / wechatIcon.getIconHeight();
        wechatIcon.setImage(wechatIcon.getImage().getScaledInstance((int)(wechatIcon.getIconWidth() * scale), 400, Image.SCALE_DEFAULT));
        wechatLabel = new JLabel(wechatIcon);
        wechatLabel.setVisible(true);
    }
}

File: easydoc-kdoc/src/main/kotlin/com/star/easydoc/kdoc/view/inner/SupportView.kt
package com.star.easydoc.kdoc.view.inner

import com.intellij.openapi.ui.DialogWrapper
import java.awt.Image
import javax.swing.ImageIcon
import javax.swing.JComponent
import javax.swing.JLabel
import javax.swing.JPanel

/**
 * @author wangchao
 * @date 2020/08/29
 */
class SupportView : DialogWrapper(false) {
    private lateinit var alipayLabel: JLabel
    private lateinit var wechatLabel: JLabel
    private lateinit var panel: JPanel

    init {
        init()
        title = "感谢大佬！"
    }

    override fun createCenterPanel(): JComponent {
        return panel
    }

    private fun createUIComponents() {
        val alipayIcon = ImageIcon(this.javaClass.getResource("/alipay.jpg"))
        var scale = 400.0 / alipayIcon.iconHeight
        alipayIcon.image = alipayIcon.image.getScaledInstance((alipayIcon.iconWidth * scale).toInt(), 400, Image.SCALE_DEFAULT)
        alipayLabel = JLabel(alipayIcon)
        alipayLabel.isVisible = true
        val wechatIcon = ImageIcon(this.javaClass.getResource("/wechat.png"))
        scale = 400.0 / wechatIcon.iconHeight
        wechatIcon.image = wechatIcon.image.getScaledInstance((wechatIcon.iconWidth * scale).toInt(), 400, Image.SCALE_DEFAULT)
        wechatLabel = JLabel(wechatIcon)
        wechatLabel.isVisible = true
    }
}